spec/mongo/uri/srv_protocol_spec.rb

```diff
@@ -220,24 +210,6 @@
         expect { uri }.to raise_error(Mongo::Error::InvalidURI)
       end
     end
-
-    context 'mongodb+srv://example.com?w=1' do
```

Comment:
Just removed this test, don't think this should throw under the new requirements.